### PR TITLE
Fix bug with themes switcher for mobile version

### PR DESCRIPTION
**Pull request by GorDi004**

### CODE REVIEW

File: common/components/CardOneTask/index.tsx [hunk 1]
```diff
@@ -16,7 +17,76 @@ interface Props {
...
+  //   {
+  //     title: 'Дата',
+  //     dataIndex: 'deadline',
+  //     key: 'deadline',
+  //     width: '16.6%',
+  //     ellipsis: true,
+  //     sorter: (a, b) => Date.parse(a?.deadline) - Date.parse(b?.deadline),
+  //     render: (date) => moment(date).format('DD-MM hh:mm'),
+  //   },
+  // ]
```
Comment (VanyaRPP) [inline common/components/CardOneTask/index.tsx:70]: Separete component

Comment (GorDi004) [reply common/components/CardOneTask/index.tsx]: Ok

File: common/assets/features/validators.ts [hunk 2]
```diff
@@ -15,13 +15,16 @@ export const deleteExtraWhitespace = (value: string): string =>
 export const validateField = (name: string): Rule[] => {
   const required: Rule = {
     required: true,
-    message: `${name[0].toUpperCase() + name.slice(1)} is required!`,
+    message: "Назва обов'язкова!",
```
Comment (Iangyl) [inline common/assets/features/validators.ts:18]: Does it a universal method? We should be more concrete, for example: "Поле email обов'язкове!"

File: common/components/CardOneTaskTable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    {
+      title: 'Дата',
+      dataIndex: 'deadline',
+      key: 'deadline',
+      width: '16.6%',
+      ellipsis: true,
+      sorter: (a, b) => Date.parse(a?.deadline) - Date.parse(b?.deadline),
+      render: (date) => moment(date).format('DD-MM hh:mm'),
+    },
+  ]
```
Comment (Iangyl) [inline common/components/CardOneTaskTable/index.tsx]: eject to separate file, please

File: common/components/ListOneTask/index.tsx [hunk 4]
```diff
@@ -7,10 +7,11 @@ import StatusTag from '../UI/StatusTag'
 import s from './style.module.scss'
 
 interface Props {
-  task: ITask
+  task: ITask[]
 }
 
-const CardOneTaskTable: React.FC<Props> = ({ task }) => {
+const ListOneTask: React.FC<Props> = ({ task }) => {
+  // console.log(tasks)
```
Comment (Iangyl) [inline common/components/ListOneTask/index.tsx:14]: remove log

**Review by Iangyl — CHANGES_REQUESTED** [review]

**Pull request merged**